scalableminds/webknossos
# Workaround to avoid false-positive version warning

See the code comment. I'd consider the issue closed then until we get another report about this (but I'd hope that this won't happen again).

I ran into the original issue while testing #6652 (luckily I had a breakpoint in that saga from another debugging session). I could verify that the version mismatch was off by one and directly after continuing from the breakpoint in the warning-saga, the `updateVersionAction` was dispatched which led to my theory about the race condition.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- the issue was basically impossible to reproduce. introducing `sleep`s at the right spots would probably work, but then one would only verify that the described race condition is fixed (maybe others exist which we didn't see yet?)
- since only a sleep in a non-critical saga was added, I don't think much harm can be caused with this PR

### Issues:
- hopefully fixes #6503

------
(Please delete unneeded items, merge only when none are left open)
